src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -213,6 +213,8 @@ def delete_vault_or_hsm(cmd, client, resource_group_name=None, vault_name=None,
 
     assert hsm_name
     hsm_client = get_client_factory(ResourceType.MGMT_KEYVAULT, Clients.managed_hsms)(cmd.cli_ctx, None)
+    logger.warning('This command will soft delete the resource, and you will still be billed until it is purged. '
+                   'For more information, go to https://aka.ms/doc/KeyVaultMHSMSoftDelete')
```

Comment:
I am personally against short links as they are difficult to maintain (https://github.com/Azure/azure-cli/pull/14123#discussion_r446755285) and introduces inconsistencies across Azure CLI repo (https://github.com/Azure/azure-cli/search?p=3&q=aka.ms). As a user myself, I am always reluctant to open a short link without knowing the actual domain it will redirects me to (in this case https://docs.microsoft.com/).

If this is mandatory, add the group `azclisecurity` as the owner of the short link.

Reply:
The short link was specifically requested by the Keyvault team. I have added the group as the owner.